10362.xml

```diff
@@ -0,0 +1,91 @@
+
+* Neither the name of the copyright holder nor the names of its
+  contributors may be used to endorse or promote products derived from
+  this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>Drop Sensor</Name>
+    <Description1><![CDATA[This LWM2M Object includes Drop Sensor information.]]></Description1>
+    <ObjectID>10362</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10362</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+		<Item ID="1">
+			<Name>Name</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>String</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description>
+				<![CDATA[The display name of the Drop Sensor.]]>
+			</Description>
+		</Item>
+		<Item ID="50">
+			<Name>Status</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Integer</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description><![CDATA[0:Normal, 1:Unknown, <0:Failed with error code(negative number).]]></Description>
+        </Item>
+		<Item ID="51">
+			<Name>Drop Detection</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Optional</Mandatory>
+			<Type>Boolean</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description><![CDATA[If Drop Detection is enabled or not,
+				the setting of which is a Boolean value (1,0) where 1 is Enabled and 0 is Disabled.]]></Description>
+		</Item>		
+		
+	  	<Item ID="100">
+			<Name>Drop Detection Control</Name>
+			<Operations>E</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type/>
+			<RangeEnumeration/>
+			<Units/>
+			<Description><![CDATA[Enable/Disable Drop Detection: Parameter: 1:Enable, 0:Disable]]></Description>
```

Comment:
Should this be type boolean with true/false?

Reply:
Yes, changed. Thanks.

Reply:
Incorrect Type/Operation combination - A Resource, which supports the "Execute" operation, MUST have no associated Data Type (none) encoded as an empty value in Object DDF file

Reply:
Removed, thanks.